addons/udes_security/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+from odoo import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.multi
+    def write(self, vals):
+        password = self._context.get("password")
+        if password:
+            partner_user = self.user_ids and self.user_ids[0] or False
```

Comment:
I don't think you need the False is needed here.

Reply:
Updated